<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeScreen</class>
 <widget class="QMainWindow" name="welcomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	border-radius:11px;
}

QWidget { 
        background-color: #0F111A;  /* Apply background to the main window */
		color:white;
    }
    
    /* Make QGroupBox transparent with no border, inheriting the main window's background color */
    QGroupBox { 
        border: none; 
    }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>461</width>
            <height>381</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:#303755;
border-radius:15px;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QStackedWidget" name="page1innerStack">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="page1InnerStack_page1">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="welcomeTitle">
                 <property name="text">
                  <string>Welcome To
Codenheimer</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QPushButton" name="startButton">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QWidget{
	font-size:14px;
	color: white;
	border-radius:20px;
}

QPushButton:hover{
	background:rgb(66, 93, 204);
}
</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page1InnerStack_page2">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QWidget" name="page1InnerStack_page2container" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>888</width>
                   <height>300</height>
                  </size>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="generalSettingsTitle">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="layoutDirection">
                     <enum>Qt::LeftToRight</enum>
                    </property>
                    <property name="text">
                     <string>Setup User</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_9" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_16">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="usernameTitle">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>142</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>142</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Username:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="usernameEdit">
                       <property name="minimumSize">
                        <size>
                         <width>192</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">background:#2B3663;
border-radius:7px;</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_11" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_18">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="newPasswordTitle">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>142</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>142</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>New Password:           </string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="newPasswordEdit">
                       <property name="minimumSize">
                        <size>
                         <width>100</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">background:#2B3663;
border-radius:7px;</string>
                       </property>
                       <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_3" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QPushButton" name="userUpdateButton">
                       <property name="minimumSize">
                        <size>
                         <width>49</width>
                         <height>21</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>49</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QWidget{
	font-size:14px;
	color: white;
	border-radius:5px
}
QPushButton{
	background:#2B3663;
}

QPushButton:hover{
	background:rgb(86, 99, 152);
}</string>
                       </property>
                       <property name="text">
                        <string>Save</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="showPasswordButton">
                       <property name="minimumSize">
                        <size>
                         <width>141</width>
                         <height>21</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>141</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QWidget{
	font-size:14px;
	color: white;
	border-radius:5px;
margin-left:10px;
margin-right:10px;
}
QPushButton{
	background:#2B3663;
}

QPushButton:hover{
	background:rgb(86, 99, 152);
}
</string>
                       </property>
                       <property name="text">
                        <string>Show Password</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_5"/>
      <widget class="QWidget" name="page_6"/>
      <widget class="QWidget" name="page_7"/>
      <widget class="QWidget" name="page_8"/>
      <widget class="QWidget" name="page_9"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
